docs/api/index.md

```diff
@@ -110,7 +110,6 @@ app.listen(3000);
   Application settings are properties on the `app` instance, currently
   the following are supported:
 
-  - `app.name` optionally give your application a name
```

Comment:
I think the convention might already been established. What is the point of removing it?